src/posthog-core.ts

```diff
@@ -538,6 +546,12 @@ export class PostHog {
             this.analyticsDefaultEndpoint = response.analytics.endpoint
         }
 
+        this.config.person_profiles = this._initialPersonProfilesConfig
+            ? this._initialPersonProfilesConfig
+            : response['defaultIdentifiedOnly']
+            ? 'identified_only'
+            : 'always'
```

Comment:
This is the bit that responds to a value passed from /decide. But I'm not sure if mutating the config like this is allowed....

Reply:
Thinking out loud, but it might be better to do this with `set_config`, as that's a clear place for future code to listen for this change and respond to it

Reply:
Sweet I pushed up a change to use that. Thanls!